docs/general/rendering.md

```diff
@@ -169,7 +169,7 @@ This will be rendered as:
     }
   }
 ```
-Note: the `Attributes` adapter (default) does not include a resource root.
+Note: the `Attributes` adapter (default) does not include a resource root. You also will not be able to override the 'data' root if you are using the :json_api adapter.
```

Comment:
:+1: 

> You also will not be able to override the 'data' root if you are using the :json_api adapter.

I rather reword this for two reasons:

1. Of the three adapters, only the JSON adapter has a document root that can be set via option.
  - (`root` is more the resource root, which may also be the document root.  It is tightly coupled to the internal concepts of `json_key` and `_type`. Those concepts should be unified.)
1. The JSON API spec multiple top-level nodes.  None of them are a resource root, so it doesn't make sense to pass in a root.

Still a great opportunity for clean up, just a little different than current diff! :)